README.md

```diff
@@ -1,87 +1,175 @@
+  document.getElementById('main')
+);
+```
+
+There's a small code style issue at play here: our line with the `MovieCard` component is super long, resulting in code
+that is pretty hard to read. Thankfully, we can add line breaks to make things more readable again:
+
+```js
+ReactDOM.render(
+  <MovieCard
+    title="Mad Max: Fury Road"
+    poster="http://image.tmdb.org/t/p/w342/kqjL17yufvn9OVLyXYpvtyrFfak.jpg"
+    genres={['Action', 'Adventure', 'Science Fiction', 'Thriller']} 
+  />,
+  document.getElementById('main')
+);
+```
+
+Notice how we passed in the genres as an inline array? We could also pass in variables instead, like this:
+
+```js
+const madMaxGenres = ['Action', 'Adventure', 'Science Fiction', 'Thriller'];
+
+ReactDOM.render(
+  <MovieCard
+    title="Mad Max: Fury Road"
+    poster="http://image.tmdb.org/t/p/w342/kqjL17yufvn9OVLyXYpvtyrFfak.jpg"
+    genres={madMaxGenres} 
+  />,
+  document.getElementById('main')
+);
+```
+
+This would also work for the `title` and `poster` props, but you get the idea.
+
+## Accessing props
+Now that we've passed in our props, let's change our hardcoded data in the `render()` method to make use of the props we
+pass in instead:
 
+```js
+class MovieCard extends React.Component {
   render() {
     return (
-      <div>
-        <h1>{this.props.greeting}</h1>
+      <div className="movie-card">
+        <img src={this.props.poster} alt={this.props.title} />
+        <h2>{this.props.title}</h2>
+        <small>Genres: {this.props.genres.join(', ')}</small>
       </div>
-    )
+    );
   }
-})
+}
 ```
 
-As a bonus, and since this lesson will probably be short, this might be a good
-time to introduce [ES class properties](http://babeljs.io/docs/plugins/transform-class-properties/)
+And that's all there is to it! For demonstrative purposes, let's render a movie card for Jurassic World:
 
-Students will need to add a transform to their `babelify` setup, but it should
-be pretty easy. Then they can do
+```js
+const jurassicWorldGenres = ['Action', 'Adventure', 'Science Fiction', 'Thriller'];
 
-```javascript
-const React = require('react')
+ReactDOM.render(
+  <MovieCard
+    title="Jurassic World"
+    poster="http://image.tmdb.org/t/p/w342/jjBgi2r5cRt36xF6iNUEhzscEcb.jpg"
+    genres={jurassicWorldGenres} 
+  />,
+  document.getElementById('main')
+);
+```
 
-class MyComponent extends React.Component {
-  defaultProps = {
-    greeting: "Hello, world!"
-  }
+Woohoo! We've created our first dynamic component. Good job!
+
+## Default values for props
+What if we didn't have a poster image for a movie? Ideally, we'd have a default poster image for that instead. Instead
+of passing in that default poster image in case we don't have one, we can tell update our `MovieCard` component to use
+a default value instead, if the `poster` prop was not provided. To do that, we add the `.propTypes` property to our
```

Comment:
Wondering if it makes sense to include the `.` on the `.propTypes`, since you've already indicated that it's a property...